src/bijectors/named_bijector.jl

```diff
@@ -0,0 +1,231 @@
+abstract type AbstractNamedBijector <: AbstractBijector end
+
+forward(b::AbstractBijector, x) = (rv = b(x), logabsdetjac = logabsdetjac(b, x))
+
+#######################
+### `NamedBijector` ###
+#######################
+"""
+    NamedBijector <: AbstractNamedBijector
+
+Wraps a `NamedTuple` of key -> `Bijector` pairs, implementing evaluation, inversion, etc.
+
+# Examples
+```julia-repl
+julia> using Bijectors: NamedBijector, Scale, Exp
+
+julia> b = NamedBijector((a = Scale(2.0), b = Exp()));
+
+julia> x = (a = 1., b = 0., c = 42.);
+
+julia> b(x)
+(a = 2.0, b = 1.0, c = 42.0)
+
+julia> (a = 2 * x.a, b = exp(x.b), c = x.c)
+(a = 2.0, b = 1.0, c = 42.0)
+```
+"""
+struct NamedBijector{names, Bs<:NamedTuple{names}} <: AbstractNamedBijector
+    bs::Bs
+end
+
+@inline names_to_bijectors(b::NamedBijector) = b.bs
```

Comment:
I'm a bit surprised if `@inline` would be needed here? Seems like something that the compiler should inline anyway, I guess?

Reply:
100% :+1: Left-over from me with worse understanding of Julia:)

Reply:
```suggestion
names_to_bijectors(b::NamedBijector) = b.bs
```